### DISCUSSION TITLE
Add Search bar for SkinnableSprite

### DISCUSSION THREAD

**Discussion by SK-Fast** [ppy/osu#21329] [Ideas] [answered]
Finding the texture I needed on this was painful. There should be a search bar so the texture can be found easier.

![image](https://user-images.githubusercontent.com/60780448/202899544-f1e012f8-6067-4a2d-8f20-d256f5f09dea.png)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/17020
